pages/game.tsx

```diff
@@ -1,24 +1,67 @@
-import React from "react";
-import Link from "next/link";
-import { Container } from "react-bootstrap";
+import Board from "../components/Board";
+import Keyboard from "../components/Keyboard";
+import { createContext, useState } from "react";
+import { boardDefault } from "../components/words";
 
-export default function game() {
-    return (
-        <Container>
-            <h1>Hello, world!</h1>
-            <Link href="/about">About</Link>
-            <h1 class="text-centered">
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-                ⬜⬜⬜⬜⬜ <br></br>
-            </h1>
-            <h2 class="text-centered">
-                Q W E R T Y U I O P <br></br>
-                A S D F G H J K L <br></br>
-                Z X C V B N M <br></br>
-            </h2>
-        </Container>
-    );
+export const GameContext = createContext(null);
```

Comment:
Nice use of React Context ... consider adding some more comments here to explain why this is necessary / what is going on, and also consider moving the context to another file since it can technically be used as a React component.